src/main/java/org/broadinstitute/hellbender/tools/diagnostics/PrintBGZFBlockInformation.java

```diff
@@ -0,0 +1,189 @@
+package org.broadinstitute.hellbender.tools.diagnostics;
+
+import htsjdk.samtools.util.BlockCompressedStreamConstants;
+import htsjdk.tribble.AbstractFeatureReader;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.argparser.ExperimentalFeature;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import picard.cmdline.programgroups.DiagnosticsAndQCProgramGroup;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * A diagnostic tool that prints information about the compressed blocks in a BGZF format file,
+ * such as a .vcf.gz file.
+ *
+ * The output looks like this:
+ *
+ * Block at file offset 0
+ *     - compressed size: 12932
+ *     - uncompressed size: 65280
+ *
+ * Block at file offset 12932
+ *     - compressed size: 9978
+ *     - uncompressed size: 65280
+ * ...
+ * etc.
+ *
+ * The output can be redirected to a file using the -O option.
+ */
+@ExperimentalFeature
+@CommandLineProgramProperties(
+        summary = "Print information about the compressed blocks in a BGZF format file",
+        oneLineSummary = "Print information about the compressed blocks in a BGZF format file",
```

Comment:
Maybe expand this comment to more explicitly summarize the compressed and uncompressed size.

Reply:
Not necessary I think -- the tool-level docs have that information.